openhands/server/routes/trajectory.py

```diff
@@ -22,9 +22,7 @@ async def get_trajectory(request: Request):
         events.
     """
     try:
-        async_stream = AsyncEventStreamWrapper(
-            request.state.conversation.event_stream, filter_hidden=True
-        )
+        async_stream = AsyncEventStreamWrapper(request.state.conversation.event_stream)
```

Comment:
Reverting this change since I am not sure if this is a proper fix. I do notice that in 

`async def close(self) -> None:`

function, we also do `filter_hidden=True`, so it's better keep the behavior consistent.

I'll defer investigation of this issue and it's anyways out of scope of this PR.